host_cleanup.sh

```diff
@@ -28,3 +28,12 @@ sudo virsh net-destroy baremetal
 sudo virsh net-undefine baremetal
 sudo virsh net-destroy provisioning
 sudo virsh net-undefine provisioning
+
+# Remove local changes made to baremetal-operator code
+eval "$(go env)"
+pushd "$GOPATH/src/github.com/metalkube/baremetal-operator"
+# Another way to cleanup would be to "rm -rf baremetal-operator"
+git checkout master
+git checkout -- deploy/role_binding.yaml
+git checkout -- deploy/service_account.yaml
+popd
```

Comment:
If you follow my suggestion of copying these files instead of modifying the contents of the git repo, you could drop these changes.

Reply:
Yes, making that changes that were suggested for 03_ocp_repo_sync.sh.